### PR TITLE
Dash live model training

### PR DESCRIPTION
**Pull request by christopherjeon**
Issue for app: #33 

# App pull request
- [ ] This is a new app
- [x] I am improving an existing app (redesigns/code "makeovers")

## About

- Playground deployment URL (new version):  https://dash-playground.plotly.host/dash-live-model-training/

## Workflow

- [x] I have created a branch in the appropriate monorepo, and the
  elements necessary for successful deployment are in place.
- [x] If the app is a redesigned and/or restyled version of an
  existing gallery app, I've summarized the changes requested in the
  appropriate Streambed issue and confirm that they have been applied.
- [x] If the app is on the Dash Gallery portal, I have added a link to
  the GitHub repository for the source code in the portal description.
- [x] I have removed *all Google Analytics code* from the app's
  `assets/` folder.

## The pre-review review

I have addressed all of the following questions:

- [x] Does everything in my code serve some purpose? (I have removed
  any dead and/or irrelevant code.)
- [x] Does everything in my code have a clear purpose? (My code is
  readable and, where it isn't, it has been commented appropriately.)]
- [x] Am I reinventing the wheel? (I have used appropriate packages to
  lessen the volume of code that needs to be maintained.)

### CODE REVIEW

File: apps/dash-live-model-training/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+dash==0.42.0
```
Comment (shammamah) [inline apps/dash-live-model-training/requirements.txt]: Please update to `dash-1.0.0`!

**Comment by celinehuang** [comment]
image
 I think this is in-between mobile and desktop version, looks kinda odd so either keep it mobile or desktop

**Comment by celinehuang** [comment]
image
 1. Maybe reduce the margin on the right side and make the fonts bigger
image
2. Make sure the mobile layout works for all mobile version, this is the IPhone 5/SE
image

File: apps/dash-live-model-training/assets/base.css [hunk 2]
```diff
@@ -0,0 +1,658 @@
...
+- Clearing
+- Media Queries
+*/
+
+/* Banner
+–––––––––––––––––––––––––––––––––––––––––––––––––– */
+.banner {
+  height: 75px;
+  background-color: #191e27;
+  /* border-radius: 5px 5px 2px 2px; */
```
Comment (celinehuang) [inline apps/dash-live-model-training/assets/base.css]: remove unnecessary comments!

File: apps/dash-live-model-training/assets/base.css [hunk 3]
```diff
@@ -0,0 +1,658 @@
...
+  #title {
+    width: 60%;
+  }
+
+  #plotly-logo {
+    padding: 5px 10px;
+    width: 20%;
+  }
+
+  /* #radio-display-mode-accuracy {
```
Comment (celinehuang) [inline apps/dash-live-model-training/assets/base.css]: delete this please!

File: apps/dash-live-model-training/assets/live-model-training.css [hunk 4]
```diff
@@ -0,0 +1,7 @@
+.markdown img {
+    width: 100%;
+}
+
+._dash-undo-redo {
```
Comment (celinehuang) [inline apps/dash-live-model-training/assets/live-model-training.css]: this is not needed after the update to ```dash==1.0.0```

File: apps/dash-live-model-training/README.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
+# Dash Live Model Training Viewer
+
+![GitHub](https://img.shields.io/github/license/mashape/apistatus.svg)
+[![GitHub stars](https://img.shields.io/github/stars/plotly/dash-live-model-training.svg)](https://github.com/plotly/dash-live-model-training/stargazers)
+
+This is a demo of the Dash interactive Python framework developed by [Plotly](https://plot.ly/).
+
+Dash abstracts away all of the technologies and protocols required to build an interactive web-based application and is a simple and effective way to bind a user interface around your Python code. To learn more check out our [documentation](https://plot.ly/dash).
+
+Try out the [demo app here](https://dash-live-model-training.plot.ly/).
```
Comment (celinehuang) [inline apps/dash-live-model-training/README.md]: I think this is a dead link!

File: apps/dash-live-model-training/app.py [hunk 6]
```diff
@@ -0,0 +1,567 @@
...
+            # Empty child function for the callback
+            html.Div(id="demo-explanation", children=[])
+        ),
+        # Body
+        html.Div(
+            className="container",
+            style={"padding": "35px 25px"},
+            children=[
+                # Hidden Div that will store the result of simulating a model run
+                # html.Div(id="storage-simulated-run", style={"display": "none"}),
```
Comment (celinehuang) [inline apps/dash-live-model-training/app.py]: remove comments please!

File: apps/dash-live-model-training/app.py [hunk 7]
```diff
@@ -0,0 +1,567 @@
...
+                                        ),
+                                    ],
+                                ),
+                            ],
+                        )
+                    ],
+                ),
+                dcc.Interval(id="interval-log-update", n_intervals=0),
+                # Hidden Div Storing JSON-serialized dataframe of run log
+                # html.Div(id="run-log-storage", style={"display": "none"}),
```
Comment (celinehuang) [inline apps/dash-live-model-training/app.py]: same thing here!

File: apps/dash-live-model-training/assets/live-model-training.css (hunk 4 above)
Comment (christopherjeon) [reply apps/dash-live-model-training/assets/live-model-training.css]: are you referring to the entire file or the `_dash-undo-redo` part?

Comment (shammamah) [reply apps/dash-live-model-training/assets/live-model-training.css]: Just the `_dash-undo-redo` part! That button is hidden by default.

File: apps/dash-live-model-training/app.py [hunk 8]
```diff
@@ -0,0 +1,566 @@
...
+                                                        "label": "MNIST",
+                                                        "value": "mnist",
+                                                    },
+                                                    {
+                                                        "label": "Fashion MNIST",
+                                                        "value": "fashion",
+                                                    },
+                                                ],
+                                                placeholder="Select a demo dataset",
+                                                searchable=False,
```
[the remark this answers is not in the document]
Comment (shammamah) [reply apps/dash-live-model-training/app.py]: ^ 👍 to this comment!

File: apps/dash-live-model-training/README.md [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+![GitHub](https://img.shields.io/github/license/mashape/apistatus.svg)
+[![GitHub stars](https://img.shields.io/github/stars/plotly/dash-live-model-training.svg)](https://github.com/plotly/dash-live-model-training/stargazers)
+
+This is a demo of the Dash interactive Python framework developed by [Plotly](https://plot.ly/).
+
+Dash abstracts away all of the technologies and protocols required to build an interactive web-based application and is a simple and effective way to bind a user interface around your Python code. To learn more check out our [documentation](https://plot.ly/dash).
+
+Try out the [demo app here](https://dash-gallery.plotly.host/dash-live-model-training/).
+
+![animated1](images/animated1.gif)
```
Comment (shammamah) [inline apps/dash-live-model-training/README.md]: Make sure this is updated!

File: apps/dash-live-model-training/demo.md [hunk 10]
```diff
@@ -0,0 +1,38 @@
+## Getting Started with the Demo
```
Comment (shammamah) [inline apps/dash-live-model-training/demo.md]:
```suggestion
## Getting Started With the Demo
```

File: apps/dash-live-model-training/app.py (hunk 8 above)
[the remark this answers is not in the document]
Comment (christopherjeon) [reply apps/dash-live-model-training/app.py]: I set the default values to MNIST and Simple Conv Net now

File: apps/dash-live-model-training/README.md (hunk 9 above)
Comment (christopherjeon) [reply apps/dash-live-model-training/README.md]: updated!

**Pull request merged**